bedrock/firefox/views.py

```diff
@@ -443,7 +443,10 @@ def hello(request):
         'zh-TW': 'https://videos.cdn.mozilla.net/uploads/FirefoxHello/firefoxhello_intro_chinese'
     }
 
-    return l10n_utils.render(request, 'firefox/hello/index.html',
+    if (waffle.switch_is_active('firefox-hello-2016')):
```

Comment:
I didn't mean schedule it, simply create the switch. Thought this was a thing that needed doing, but if I'm incorrect then carry on :smile: